app/views/AgentForOrgCheckYourAnswersView.scala.html

```diff
@@ -77,6 +77,14 @@ <h1 class="govuk-heading-l">@(messages("checkYourAnswersForAgents.heading"))</h1
 @govukButton(
 ButtonViewModel(messages("checkYourAnswersForAgents.button.continue"))
 )
+ @govukButton(
```

Comment:
This block of code appears 3 times in 3 different files. I think it's worth wrapping it up in a method like _@saveButtons_ and just use it wherever we need it.

Reply:
Done :)